src/OutSystems/GridAPI/Selection.ts

```diff
@@ -181,12 +182,12 @@ namespace OutSystems.GridAPI.Selection {
             gridID,
             errorCode:
                 OSFramework.DataGrid.Enum.ErrorCodes.API_FailedGetSelectionMax,
-            hasValue: true,
             callback: () => {
                 const grid = GridManager.GetGridById(gridID);
 
                 return grid.features.selection.getSelectionMaxMin(true);
-            }
+            },
+            hasValue: true
```

Comment:
Not sure about here, since the max and min values can be -1 and valid, but maybe we can add for consistency,

Reply:
Same as here as the previous comment: _This one, I'm not sure, as -1 is a valid value... better would be not to return any value at all... but not possible... as such left 0 as an error value - discuss-able._